uaclient/api/u/pro/security/fix/_common/plan/v1.py

```diff
@@ -389,6 +390,7 @@ def __init__(
         self,
         *,
         title: str,
+        current_status: Optional[str],
```

Comment:
Since this is optional, lets put it down at the bottom with a default value of `None`